containers/Auth/LoginScreen.js

```diff
@@ -65,45 +69,55 @@ export const LoginScreen = () => {
         touched,
         isValid
       }) => (
-        <View style={styles.loginContainer}>
-
-          <TextInputFormField
-            field='E-mail address:'
-            name='username'
-            value={values.username}
-            placeholder=''
-            returnKeyType='next'
-            returnKeyLabel='next'
-            onChangeText={handleChange('username')}
-            onBlur={handleBlur('username')}
-            error={errors.username}
-            touched={touched.username}
-          />
-          <TextInputFormField
-            field='Password:'
-            name='password'
-            value={values.password}
-            placeholder=''
-            returnKeyType='next'
-            returnKeyLabel='next'
-            onChangeText={handleChange('password')}
-            onBlur={handleBlur('password')}
-            error={errors.password}
-            touched={touched.password}
-            secureTextEntry
-          />
-          <Button onPress={handleSubmit} title="Login" disabled={!isValid} />
+        <View style={styles.flexContainer}>
+          <View style={styles.container}>
+            <ScrollView>
```

Comment:
@phillimorland It was really that easy! Just adding the ScrollView here worked! :)